static/src/javascripts/bootstraps/enhanced/trail.js

```diff
@@ -102,8 +102,13 @@ define([
 
     function repositionComments() {
         if (!identityApi.isUserLoggedIn()) {
+            var $comments = $('.js-comments');
```

Comment:
yeah only for anything that will/could trigger layout. getting a reference to dom object is ok